bot/exts/utilities/realpython.py

```diff
@@ -88,4 +91,4 @@ async def realpython(self, ctx: commands.Context, amount: Optional[int] = 5, *,
 
 def setup(bot: Bot) -> None:
     """Load the Real Python Cog."""
-    bot.add_cog(RealPython(bot))
+    bot.add_cog(RealPython(bot))
```

Comment:
I think the linter is complaining about not having a new line at the end of the file.